my-app/src/components/ContentList.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+const ContentList = ({content}) => {
+    return (
+        <div className="cards_box">
+     { content && content.map(content=>(  
+
+          <div class="card">
```

Comment:
Oops! your `className` shouldn't be called `class` in React. As we mentioned, It's a reserved word in javascript for classes only. Under the hood, React will understand that this `className` will call a `CSS` styling class and it will be converted eventually into a `class` attribute in the DOM tree. To get it more; inspect your elements in the browser and check how your `div` in line 4  would look like.